hitcon-ctf-2016/pwn/heart-attack-400/README.md

```diff
@@ -14,4 +14,4 @@
 
 ## Other write-ups and resources
 
-* none yet
+* [Writeup by the challenge author](https://github.com/mehQQ/public_writeup/tree/master/hitcon2016/HeartAttack)
```

Comment:
May I suggest
`[NAME (Challenge Author)](https://github.com/mehQQ/public_writeup/tree/master/hitcon2016/HeartAttack)`
where you replace `NAME` with your own?